packages/dropdown/src/Dropdown.tsx

```diff
@@ -92,7 +94,9 @@ const Dropdown = <T extends string | object | number>(props: DropdownProps<T>) =
   const parseChildren = () => {
     const [targetChild, contentChild] = Children.toArray(children);
     return {
-      target: targetChild as ReactElement,
+      target: targetChild as FunctionComponentElement<
+        AriaAttributes & { ref: ForwardedRef<HTMLElement | undefined>; isopen: string }
+      >,
```

Comment:
Can you explain what the `isopen` is? It's a bit confusing since we have `isOpen` as well. Thanks!

Reply:
It's a prop for the underlying `PopoverTarget`, I believe.

Reply:
Ah, I see. Thanks!

Reply:
Is it supposed to be a boolean? (looking at `Popover.tsx, line 86`

Reply:
yea, this looks strange to me too. Why was this needed?

Reply:
The prop was being passed to the element clone anyway. `ReactElement` doesn't include a `ref` property, so I needed to cast `target` to something else, and based on what I saw in the react types, `FunctionComponentElement` seemed to fit. `ReactElement` has `any` as the default value of its props generic, but `FunctionComponentElement` has no default props generic value, so I needed to specify that `isopen` was a prop we expect, along with `AriaAttributes`, because we provide those props to the element when we clone it.

Reply:
> Is it supposed to be a boolean? (looking at `Popover.tsx, line 86`

No idea. I'm guessing that bit is very old code that's been preserved into launchpad. but yeah, it's actually supplying a `string` instead, and that seemed weird to me, but I didn't want to mess with things that didn't need messing. Other than the `mergeRefs` thing here, I've only altered typings to fit.